asciidoc/courses/gds-community-detection/modules/2-preprocess/lessons/3-c-categorical/reset.cypher

```diff
@@ -1,10 +1,17 @@
-// Delete All nodes
-MATCH (p:Person) DETACH DELETE p;
+// reload data
+:auto LOAD CSV 
```

Comment:
`:auto` is a command to tell Browser to run the query in an auto-commit transaction. 

Thinking about it, you should be able to do it without the subquery:

```
LOAD CSV
WITH HEADERS
FROM "https://raw.githubusercontent.com/tomasonjo/blog-datasets/main/survey/responses.csv" AS row

CREATE (p:Person)
SET p = row
WITH p
UNWIND keys(p) AS key
WITH p, key, toFloat(p[key]) AS value
WHERE value IS NOT null
CALL apoc.create.setProperty(p, key, value) YIELD node
RETURN DISTINCT 'done' AS result
```